### PR TITLE
Remove unused setter calls in map->event-policy

### PR DESCRIPTION
**Pull request by predar**
Fix for issue 44 #44 @barkanido

### CODE REVIEW

File: test/aerospike_clj/policy_test.clj [hunk 1]
```diff
@@ -0,0 +1,30 @@
+(ns aerospike-clj.policy-test
+  (:require [clojure.test :refer [deftest is]]
+            [aerospike-clj.policy :as policy])
+  (:import [com.aerospike.client.async EventPolicy]))
+
+(defn- verify-event-policy-properties
+  [^EventPolicy event-policy conf]
+  (is (= (get conf "maxCommandsInProcess" 0) (.-maxCommandsInProcess event-policy)))
```
Comment (barkanido) [inline test/aerospike_clj/policy_test.clj]: I do not think there is a need for the `.-` accessor:
```clojure
=> (import '[com.aerospike.client.async EventPolicy])
=> (def ep (EventPolicy.))
=> (.minTimeout ep)
100
```

**Review by barkanido — CHANGES_REQUESTED** [review]
Thanks a lot!

and kudos for the test addition.
overall, LGTM.

**Review by barkanido — APPROVED** [review]

**Comment by barkanido** [comment]
Thanks! merging.

**Comment by predar** [comment]
> Thanks! merging.

Great. Glad I could help. Thank you, too!

**Pull request merged**